content/en/opentelemetry/guide/semantic_mapping.md

```diff
@@ -60,10 +65,22 @@ For more information, see [Unified Service Tagging][2].
 | `k8s.cronjob.name` | `kube_cronjob` |
 | `k8s.namespace.name` | `kube_namespace` |
 | `k8s.pod.name` | `pod_name` |
+| `app.kubernetes.io/name` | `kube_app_name` |
```

Comment:
Although I agree we need to document this somewhere, I don't think this is the right place. This table is for the mapping from OTel Conventions to DD conventions, but the additions here are not OTel conventions.

Perhaps we can create a new table at the bottom where the left column name is "Kubernetes Labels" or "Kubernetes Conventions" instead of "OpenTelemetry convention".

What do you think ?

Reply:
Sounds good. Feel free to take this as the basis for a docs fix.